### PR TITLE
Doing a coding review to one of my last project

### PR DESCRIPTION
**Pull request by Jessemccallums**
This pull request has the aim of doing a code review from one of my past project in admission trials.

Project requirements

I created a new empty repo.

I copied my code from the project that you completed during the Admissions Trial.

I created a new branch for your copied code.

I opened a pull request.

I played the role of the Code Reviewer

### CODE REVIEW

**Review by Jessemccallums — COMMENTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Jessemccallums,
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 

## Required Changes :recycle: :recycle: 

- [x] Add linters, is very important to add linters to your project because it will help you to be more intelligent with your code 
- [x] Add sections to the html file, it will help you to have a more organized code.

Cheers and Happy coding!👏👏👏


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Jessemccallums , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 🔴  ⛔ 

You have submitted a project, that has no review.. please make sure you add some review inline comments and changes to be made. you are almost there 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css"  type="text/css"  />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Anonymous+Pro&display=swap" rel="stylesheet">
+</head>
+<body>
+
```
Comment (Jessemccallums) [inline index.html:14]: Add sections to your code to have it more organized

File: main.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+body {
+    background-color: cornflowerblue;
```
Comment (Jessemccallums) [inline main.css:2]: You must add RGB colors to your project, please, change it

**Review by Kandy-Peter — APPROVED** [review]
Hi @Jessemccallums ,

# APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏

Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**